CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 
 ## [Unreleased]
 
+## [5.0.1]
+
+### Added
+- Support to WFS reader using the original data from download
```

Comment:
This one is related to hale-connect. Also, this is something that cannot be tested directly in hale studio as the change cannot be triggered from UI or CLI.
Not sure if it should be mentioned in the release notes. However, I would wait for @thorsten-reitz and @stempler  feedback as well.